slim.Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM python:3.7-slim as runtime
+EXPOSE 8080
+
+WORKDIR /girder
+
+RUN apt update && apt install -qy gcc git
+COPY . /girder/
+
+RUN pip install --upgrade --upgrade-strategy eager .
```

Comment:
Add `--no-cache-dir` so packages aren't cached.

Reply:
This should probably go in the regular Dockerfile too?

Reply:
Yes.